src/Components/AppFooter.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import styled from "styled-components";
+
+
```

Comment:
There are some extra empty lines in this file (and in other components' files), we can remove them!

Reply:
We can add prettier in the same issue that we'll add some tests to the app. I'll update that issue soon. For now just removing the extra lines is fine!